lib/websocket.js

```diff
@@ -82,6 +180,54 @@ function init (env, ctx, server) {
       console.info('emitted announcement to all clients');
     }
   };
+  
+  websocket.authorize = function authorize(data, callback) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['callback' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1644868632/issues/source?bid=2452379&fileBranchId=3019946#l184)